index.js

```diff
@@ -149,10 +155,15 @@ SQLdown.prototype._get = function (key, options, cb) {
     if (!res.length) {
       return cb(new Error('NotFound'));
     }
+
     try {
       var value = JSON.parse(res[0].value);
       if (asBuffer) {
-        value = new Buffer(value);
+        if(value.type === 'Buffer'){
```

Comment:
@kumavis whether we store the buffers as base64 encoded strings or UTF strings is a seperate issue which I'm happy to discuss but I'd like to focus this issue on fixing the broken buffers.